concent_api/conductor/service.py

```diff
@@ -0,0 +1,51 @@
+import logging
+
+from django.db.models import Q
+
+from common.constants import ErrorCode
+from conductor.exceptions import VerificationRequestAlreadyInitiatedError
+from conductor.models import ResultTransferRequest
+from conductor.models import UploadReport
+from conductor.models import VerificationRequest
+from core.tasks import result_upload_finished
+
+
+logger = logging.getLogger(__name__)
+
+
+def report_finished_upload(file_path: str, result_transfer_request: ResultTransferRequest) -> None:
```

Comment:
I have been thinking about it for a while and the name doesn't seem right. 'finished_upload' is reported conditionally. What always happens in this function is updating `UploadReport`. Thus maybe this function should be renamed to `update_upload_report`? I dnon't know. something bothers me here, but I am not sure what